src/main/java/picard/vcf/VcfUtils.java

```diff
@@ -1,15 +1,16 @@
 package picard.vcf;
 
+import htsjdk.samtools.util.IOUtil;
 import java.io.File;
 
 /**
  * Created by farjoun on 4/1/17.
  */
 public class VcfUtils {
 
-    static String UNCOMPRESSED_VCF_ENDING = ".vcf";
-    static String COMPRESSED_VCF_ENDING = ".vcf.gz";
-    static String BCF_ENDING = ".bcf";
+    public static String UNCOMPRESSED_VCF_ENDING = IOUtil.VCF_FILE_EXTENSION;
+    public static String COMPRESSED_VCF_ENDING = IOUtil.COMPRESSED_VCF_FILE_EXTENSION;
+    public static String BCF_ENDING = IOUtil.BCF_FILE_EXTENSION;
```

Comment:
Why define these variables at all, if we could just use `IOUtil.VCF_FILE_EXTENSION` etc. directly?